src/components/Sidebar/index.js

```diff
@@ -20,6 +20,7 @@ const CustomSidebar = ({location, user, articles}) => {
       <Menu theme="dark" mode="inline" defaultSelectedKeys={[location.pathname]} style={{height: '100%', borderRight: 0, marginTop: '20px'}}>
         {user.username && user.isContributor && <Menu.Item key="/mycontributions"><Link to="/mycontributions">My Contributions</Link></Menu.Item>}
         {/*{user.isModerator && */}<Menu.Item key="/review"><Link to="/review">Review</Link></Menu.Item>{/*}*/}
+        {/*{user.isSupervisor && */}<Menu.Item key="/users"><Link to="/users">Users</Link></Menu.Item>{/*}*/}
```

Comment:
We can uncomment this for production @ateufel

Reply:
Yeah, i will add it with the production flag, give me a minute!